lib/package.json

```diff
@@ -3,40 +3,40 @@
   "version": "0.0.1",
   "type": "module",
   "description": "Storyblok SDK to connect Storyblok with Svelte",
-  "main": "./dist/storyblok-svelte.js",
-  "module": "./dist/storyblok-svelte.mjs",
-  "svelte": "index.ts",
-  "types": "./dist/main.d.ts",
+  "main": "./dist/index.js",
+  "module": "./dist/index.js",
+  "svelte": "./dist/index.js",
+  "types": "./dist/index.d.ts",
   "files": [
     "dist",
-    "index.ts",
-    "types.ts",
-    "StoryblokComponent.svelte"
+    "README.md"
```

Comment:
Looks like this can safely be omitted actually - [see here](https://kit.svelte.dev/docs/packaging#anatomy-of-a-package-json-files).